data/json/items/tools.json

```diff
@@ -6661,7 +6661,7 @@
       {
         "type": "repair_item",
         "item_action_type": "repair_metal",
-        "materials": [ "kevlar", "plastic", "iron", "steel", "hardsteel", "aluminum", "copper" ],
+        "materials": [ "kevlar", "plastic", "iron", "steel", "hardsteel", "aluminum", "copper", "silver", "gold" ],
```

Comment:
As I pointed in https://github.com/CleverRaven/Cataclysm-DDA/pull/22112#issuecomment-335064701, soldering iron cannot achieve temperature to melt down gold.

Reply:
then It shouldnt repair iron either

Reply:
IMO the repair by soldering iron is to use molten solder "glue" broken parts together.

Reply:
> IMO the repair by soldering iron is to use molten solder "glue" broken parts together.

Yes, that's why iron is repairable by soldering iron.

Reply:
I have just checked, you can use solder mixed with brass/silver dust to repair your jewelry. http://m.1688.com/offer/522837469084.html?spm=a26g8.7662792.1998744630.2

Reply:
>I have just checked, you can use solder mixed with brass/silver dust to repair your jewelry. 

Jewelry made of what?

Reply:
gold/silver, see the table in this page.
http://www.mining120.com/tech/show-htm-itemid-27135.html

Reply:
Check a bit more, you are right, soldering iron cant use silver solder.